packages/palette/src/elements/Marquee/Marquee.story.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { Marquee } from "./Marquee"
+
+export default {
+  title: "Components/Marquee",
+}
+
+export const Default = () => {
+  return <Marquee marqueeText="Black Owned" />
+}
+
+export const LightVariant = () => {
+  return <Marquee marqueeText="Black Owned" variant="defaultLight" />
+}
+
+export const Brand = () => {
+  return <Marquee marqueeText="Black Owned" variant="brand" />
+}
+
+export const NoDivider = () => {
+  return <Marquee marqueeText="Black Owned" noDivider />
+}
+
+export const ExtraFast = () => {
+  return <Marquee marqueeText="Black Owned" speed="5s" />
+}
+
+export const ExtraSlow = () => {
+  return <Marquee marqueeText="Black Owned" speed="20s" />
+}
```

Comment:
Not at all blocking but you could use our `States` component to make this a little less verbose and easier to read for consumers.

https://github.com/artsy/palette/blob/03efe61df5146dbdfaebeb1b778c6827efba4ddf/packages/palette/src/elements/Button/Button.story.tsx#L15-L26

Outputs:

![](https://static.damonzucconi.com/_capture/ljMA3GYa.png)